src/requests/TrainingRequests.ts

```diff
@@ -69,21 +88,28 @@ export class ListTrainingRequests extends HTTPGetRequest {
                         )}`,
                     );
                 }
+
+                this.filterQuery.disabledReasons.push(disabledReasonString);
             });
         }
+
         if (this._req.query["wings"]) {
             const wingStrings: string[] = this._req.query["wings"]
                 .toString()
                 .toLowerCase()
                 .split(",");
             wingStrings.forEach((wingString) => {
-                if (!Number.parseInt(wingString)) {
+                const wingNum = Number.parseInt(wingString);
+                if (!wingNum) {
+                    // todo: this should never fire unless wingNum equals 0 as parseInt raises an error with non-numerics
```

Comment:
Doesn't parseInt just set the value to NaN? Come to think of it this should probably read as

```ts
if (Number.isNaN(wingNum)) {

}
```